canonical-web-and-design/multipass.run
# Fix the version selection to be the latest release

## Done
Switched from checking the committed branch to the draft flag.

## QA
- Open the homepage
- Scroll down to the download button
- Check the Windows and Mac links are to v7.0 

## Details
Fixes https://github.com/canonical-web-and-design/multipass.run/issues/38
